tests/tests_integration/test_api/test_hosted_extractors/test_jobs.py

```diff
@@ -0,0 +1,163 @@
+from __future__ import annotations
+
+import pytest
+
+from cognite.client import CogniteClient
+from cognite.client.data_classes.hosted_extractors import (
+    CogniteFormat,
+    Destination,
+    DestinationWrite,
+    EventHubSourceWrite,
+    Job,
+    JobList,
+    JobLogsList,
+    JobMetricsList,
+    JobUpdate,
+    JobWrite,
+    SessionWrite,
+    Source,
+)
+from cognite.client.exceptions import CogniteAPIError
+from cognite.client.utils._text import random_string
+
+
+@pytest.fixture
+def fresh_session(cognite_client: CogniteClient) -> SessionWrite:
+    new_session = cognite_client.iam.sessions.create(session_type="ONESHOT_TOKEN_EXCHANGE")
+    yield SessionWrite(nonce=new_session.nonce)
+    cognite_client.iam.sessions.revoke(new_session.id)
```

Comment:
Let's move this to `conftest` since it seems to be reused in many hosted extractors sub-api tests